java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogSnapshotStack.java

```diff
@@ -155,7 +158,7 @@ private void createSnapshotDeletionLambda(InstanceProperties instanceProperties,
                 .runtime(Runtime.JAVA_11)
                 .handler("sleeper.statestore.snapshot.TransactionLogSnapshotDeletionTriggerLambda::handleRequest")
                 .environment(Utils.createDefaultEnvironment(instanceProperties))
-                .reservedConcurrentExecutions(1)
+                .reservedConcurrentExecutions(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_LAMBDA_CONCURRENCY_RESERVED))
```

Comment:
This should be set on the deletion function. The trigger function needs this set to 1.